### PR TITLE
NATIVE-604: Add release notes docu

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+[Steffen](mailto:kleinle@integreat-app.de) or [our info mail](mailto:info@integreat-app.de) and visit our [website](https://integreat-app.de).
+
+Other team members you can contact are 
+[Lutz](mailto:lonnemann@integreat-app.de)
+, [Sarah](mailto:sporck@integreat-app.de)
+, [Michi](mailto:markl@integreat-app.de)
+, [Simon](mailto:vortkamp@integreat-app.de)
+, [Nick](mailto:klug@integreat-app.de)
+, [Anja](mailto:stricker@integreat-app.de)
+and [Sven](mailto:seeberg@integreat-app.de).
```
Comment (Taggotty) [inline docs/CONTRIBUTING.md:18]:
```suggestion
[Lutz](mailto:lonnemann@integreat-app.de),
 [Sarah](mailto:sporck@integreat-app.de), 
 [Michi](mailto:markl@integreat-app.de), 
 [Simon](mailto:vortkamp@integreat-app.de), 
 [Nick](mailto:klug@integreat-app.de), 
 [Anja](mailto:stricker@integreat-app.de)
and [Sven](mailto:seeberg@integreat-app.de).
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+**Creating a pull request from a fork prevents checks from the CI. It is a good way to make contact though.**
+
+### Release notes
+
+Pull requests with changes that are visible to our end users should always include one (or more) release note describing the changes.
+For other changes, these release notes are optional, but make sure to set the flag `show_in_stores` to false.
+
+To add a release note:
+* Have a look at the [template](../release-notes/ReleaseNoteTemplate.yml).
+* Add a new release note to the [unreleased directory](../release-notes/unreleased).
```
Comment (Taggotty) [inline docs/CONTRIBUTING.md:40]: I guess the folders not yet exist, right?

Comment (klinzo) [reply docs/CONTRIBUTING.md:40]: Yes, because there were no release notes until yet. The unreleased dir should exist though.

**Review by maxammann — APPROVED** [review]
I added the corresponding item to the DoD

**Comment by maxammann** [comment]
ready for merge

**Pull request merged**